compiler/luci-interpreter/include/luci_interpreter/core/Tensor.h

```diff
@@ -162,6 +162,8 @@ class Tensor
 
   void set_offset(int32_t offset) { _offset = offset; }
 
+  void change_data_type(DataType element_type) { _element_type = element_type; }
```

Comment:
(optional) Personally, I think better to name this method `set_data_type` in accordance with other methods like `set_observable` or `set_data_buffer`.